src/main/java/org/opensearch/security/securityconf/impl/SecurityDynamicConfiguration.java

```diff
@@ -322,7 +314,19 @@ public boolean containsAny(SecurityDynamicConfiguration other) {
 
     public boolean isHidden(String resourceName) {
         final Object o = centries.get(resourceName);
-        return o != null && o instanceof Hideable && ((Hideable) o).isHidden();
```

Comment:
TIL that a nullcheck is redundant if calling instanceof immediately after because `null instanceof SomeClass ` will always return false.

Reply:
yes